File: src/Backend/Curotec.Store/backend/Curotec.Store.Business/Models/Contractor.cs
namespace Curotec.Store.Business.Models
{
    public class Contractor : Entity
    {
        public string Name { get; set; } = string.Empty;
        public string Document { get; set; } = string.Empty;
        public Address Address { get; set; } = default!;
        public bool Active { get; set; }

        /* EF Relations */
        public IEnumerable<Product> Products { get; set; } = default!;
    }
}

File: src/FrontEnd/Curotec-Web/src/app/contractor/models/contractor.ts
import { Address } from './address';
//import { Product } from 'src/app/product/models/product';

export class Contractor {
  id: string = '';
  name: string = '';
  document: string = '';
  active: boolean = false;
  address: Address = new Address();
  products: [] | any;
}
